engine/src/main/java/org/terasology/world/chunks/internal/ChunkImpl.java

```diff
@@ -346,8 +346,7 @@ public int chunkToWorldPositionZ(int z) {
     public AABBfc getAABB() {
         if (!aabb.isValid()) {
             org.joml.Vector3f min = JomlUtil.from(getChunkWorldOffset().toVector3f());
-            org.joml.Vector3f max = new org.joml.Vector3f(Chunks.CHUNK_SIZE);
-            max.add(min);
+            org.joml.Vector3f max = min.add(new org.joml.Vector3f(Chunks.CHUNK_SIZE));
```

Comment:
I must have been drunk when suggestion and commiting this 🙈 This obviously changes the value of `min` instead of computing a new `max` ... 🤦 

```suggestion
            org.joml.Vector3f max = new org.joml.Vector3f(Chunks.CHUNK_SIZE).add(min);
```